lib/model.js

```diff
@@ -3503,6 +3507,40 @@ Model.applyDefaults = function applyDefaults(doc) {
   return doc;
 };
 
+/**
+ * Apply this model's virtuals to a given POJO. Virtuals execute with the POJO as the context `this`.
+ *
+ * #### Example:
+ *
+ *     const userSchema = new Schema({ name: String });
+ *     userSchema.virtual('upper').get(function() { return this.name.toUpperCase(); });
+ *     const User = mongoose.model('User', userSchema);
+ *
+ *     const obj = { name: 'John' };
+ *     User.applyVirtuals(obj);
+ *     obj.name; // 'John'
+ *     obj.upper; // 'JOHN', Mongoose applied the return value of the virtual to the given object
+ *
+ * @param {Object} obj object or document to apply virtuals on
```

Comment:
```suggestion
 * @param {Object} doc object or document to apply virtuals on
```

consistency with the parameter name